src/test/java/apoc/export/cypher/ExportCypherTest.java

```diff
@@ -29,6 +30,7 @@
 
     private static final Map<String, Object> exportConfig = Collections.singletonMap("separateFiles", true);
     private static GraphDatabaseService db;
+    private static GraphDatabaseService dbEnterprise;
```

Comment:
Usare la rule TestName per attivare il db enterprise in base al nome del metodo che dovrà avere come suffisso "WithEnterpriseDB", a questo punto rimuovere la variable `dbEnterprise` e usare solo `db` per entrambi i casi